ideascube/serveradmin/tests/test_views.py

```diff
@@ -642,3 +642,46 @@ def test_staff_forgets_non_existing_connection(mocker, staffapp):
     assert u'No known Wi-Fi networks' not in res.unicode_body
     assert u'my home network' in res.unicode_body
     assert u'random open network' in res.unicode_body
+
+
+def test_staff_should_see_packages_with_cards(staffapp, systemuser, catalog):
+    from ideascube.configuration import set_config
+    set_config('home-page', 'displayed-package-ids', ['test.package1'], systemuser)
+
+    catalog.create_package(id = 'test.package1',
+                           name='Test package1',
+                           description='Test package1 description',
+                           language='fr',
+                           staff_only=False)
+    catalog.create_package(id='test.package2',
+                           name='Test package2',
+                           description='Test package2 description',
+                           language='fr',
+                           staff_only=False)
+    catalog.create_package(id='test.package3',
+                           name='Test package3 no template',
+                           description='Test package3.\n'
+                                       'A package without template'
+                          )
+
+    with catalog:
+        res = staffapp.get(reverse("server:home_page"), status=200)
+        assert 'Test package1' in res.unicode_body
+        assert 'Test package2' in res.unicode_body
+        assert 'Test package3' in res.unicode_body
+
+        form = res.forms['cards']
+        inputs = form.fields['displayed']
+        for input_ in inputs:
+            should_be_checked = (input_._value == 'test.package1')
```

Comment:
What if no input has its value set to `test.package1`? Can you ensure there is exactly 1 which does?